index.js

```diff
@@ -4,9 +4,42 @@
 
 import 'react-native-gesture-handler';
 import {AppRegistry} from 'react-native';
+import BackgroundFetch from 'react-native-background-fetch';
+
 import App from './src/App';
 import Config from './src/CONFIG';
 import additionalAppSetup from './src/setup';
 
+
+const headlessTask = async (event) => { // eslint-disable-line
+
+    // Get task id from event {}:
```

Comment:
The task should not be defined here in `index.js`
It should be defined in `libs` and then registered during `setup` or in `Expensify.js` `didMount` phase

---

I see this is already done, and there's a lib about this, so the code here seems like a leftover that should be removed